docs/resources/Channel.md

```diff
@@ -711,6 +718,7 @@ This endpoint supports requests with `Content-Type`s of both `application/json`
 | embed            | [embed](#DOCS_RESOURCES_CHANNEL/embed-object) object                       | embedded `rich` content                                 |
 | payload_json     | string                                                                     | JSON encoded body of any additional request fields.     |
 | allowed_mentions | [allowed_mentions](#DOCS_RESOURCES_CHANNEL/allowed-mentions-object) object | allowed mentions for a message                          |
+| message_reference | [message reference](#DOCS_RESOURCES_CHANNEL/message-object-message-reference-structure) | include to make your message a reply |
```

Comment:
Probably worth mentioning that this is also returned on all replies as well, regardless of whether referenced_message is returned, since this is just id's